### ISSUE TITLE
Translate the UI into French and Spanish with `i18next.js`

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1060]
This would be a milestone towards the very old issue #13. Maybe this is achievable by the Kiwix JS 10-year anniversary? At least I think Spanish is achievable (because I can do it myself).

It's a bit ambitious: the anniversary is 23rd August.... 🤔 And we wouldn't be able to get a release out by that date as well.

**Comment by kelson42** [comment]
#13 should better implemented first IMO.

**Comment by Jaifroid** [comment]
Unfortunately, Banana is not being actively maintained anymore and it doesn't have support for ES6 modules, which we are now using in this project.

So I checked out the alternatives, and found i18next.js, which uses almost exactly the same JSON format for translation strings (and is compatible with translatewiki.net).

The latest version won't work with IE11, even after bundling with rollup, but v14 works OK. It might be possible to go a bit higher too, but for now #1061 is passing tests.

**Comment by Jaifroid** [comment]
Getting there 😊:

![image](https://github.com/kiwix/kiwix-js/assets/4304337/b96add34-670c-4d4c-bb71-98f7b084556b)

**Comment by Jaifroid** [comment]
He he, even works with old Firefox OS:

![image](https://github.com/kiwix/kiwix-js/assets/4304337/295bf54b-e683-413c-9eb2-43a9125f652b)

**Comment by Jaifroid** [comment]
Nearly there (phew, this is hard work)...

![image](https://github.com/kiwix/kiwix-js/assets/4304337/d396bbea-2c57-4cff-864f-0c27ba04e6c4)

**Comment by Jaifroid** [comment]
And now we have French... 😊

![image](https://github.com/kiwix/kiwix-js/assets/4304337/a7c6746a-8e5e-4e36-afa6-1ade41372e70)
